layouts/partials/cards/default.html

```diff
@@ -15,8 +15,10 @@
     <div>
       {{ .text }}
     </div>
-    {{ if isset . "readMoreLink" }}
-      {{ partial "cards/_footer.html" . }}
-    {{ end }}
+    <div class="border-t border-card-dark pt-5 text-right">
+      {{ if isset . "readMoreLink" }}
```

Comment:
Moving the container outside this file will create an empty space when we don't render a link in the `dict`